xarray/core/dataarray.py

```diff
@@ -6636,9 +6636,10 @@ def interp_calendar(
 
     def groupby(
         self,
-        group: Hashable | DataArray | IndexVariable,
+        group: Hashable | DataArray | IndexVariable = None,
```

Comment:
```suggestion
        group: Hashable | DataArray | IndexVariable | None = None,
```
Odd, I thought we didn't allow implicit type hints?